pkg/sfu/redprimaryreceiver.go

```diff
@@ -88,11 +92,13 @@ func (r *RedPrimaryReceiver) ForwardRTP(pkt *buffer.ExtPacket, spatialLayer int3
 
 		// not modify the ExtPacket.RawPacket here for performance since it is not used by the DownTrack,
 		// otherwise it should be set to the correct value (marshal the primary rtp packet)
-		count += r.downTrackSpreader.Broadcast(func(dt TrackSender) {
-			_ = dt.WriteRTP(&pPkt, spatialLayer)
+		r.downTrackSpreader.Broadcast(func(dt TrackSender) {
+			if dt.WriteRTP(&pPkt, spatialLayer) {
+				writeCount.Inc()
+			}
 		})
```

Comment:
This is a bit off. This could generate more than one packet from a RED packet, but forwarding latency will be recorded as single packet. So, it will have a higher value of forwarding latency. It will skew the latency calculation towards higher value a bit, but this use case is small and should have negligible impact on measurement.